f8a_worker/workers/graph_populator.py

```diff
@@ -0,0 +1,242 @@
+import logging
+import re
+import time
+from datetime import datetime
+
+logger = logging.getLogger(__name__)
+
+# Compile patterns to clean a string (used for safety against GraphQL injection attack)
+danger_word_list = ["drop\(\)", "V\(\)", "count\(\)"]
```

Comment:
some test for these patters would be nice

Reply:
good suggestion